build.sh

```diff
@@ -124,6 +125,25 @@ build_android() {
   (cd bindings/bdk-kotlin && ./gradlew :android:build)
 }
 
+copy_lib_python() {
+  echo -n "Copy "
+  case $OS in
+    "Darwin")
+      echo -n "Darwin "
+      mkdir -p bindings/bdk-python/test/
+      cp target/debug/libbdkffi.dylib bindings/bdk-python/test/
+      ;;
+    "Linux")
+      echo -n "linux" ;;
+  esac
+  echo "library to python sub-project"
+}
+
+build_python() {
+  copy_lib_python
+  uniffi-bindgen generate src/bdk-python.udl --no-format --out-dir bindings/bdk-python/test/ --language python
```

Comment:
Shouldn't the output be to the `bindings/bdk-python/src` folder and then imported by your test?

Some info in the ["Packaging Python Projects"](https://packaging.python.org/tutorials/packaging-projects/) docs.

Reply:
Yes this is a good idea. I'm a bit rusty on my Python and wanted to commit something that was running, but I'll take a look at cleaning up the directory!